helm/sematic-server/values.yaml

```diff
@@ -21,6 +18,10 @@ database:
   url: postgres://postgresql:5432/postgres # REPLACE ME
 
 deployment:
+  socket_io:
+    dedicated: false
+    address: https://my.sematic-socketio # REPLACE ME
+  worker_count: 1 # if you want to increase this, enable socket_io.dedicated above
```

Comment:
another reason to want it always enabled ;-)

Reply:
Well we don't necessarily want to always deploy 2 instances.

Reply:
One should do, it's true. But the overhead of us having to care about that case is not worth it IMHO.

Reply:
Set the value of `dedicated` to `true` after offline discussion.